spec/datadog/core/error_spec.rb

```diff
@@ -184,7 +184,7 @@ def message; end
         end
 
         context 'benchmark' do
-          before { skip('Benchmark results not currently captured in CI') if ENV.key?('CI') }
+          before { skip('Benchmark not run by default') }
```

Comment:
So they're always disabled? Should we have a `BENCHMARK` env var?

Reply:
I wouldn't mind it, but to be honest I'm not sure how often it would be used. So I'd suggest waiting until we really want to do something with these benchmarks.